### PR TITLE
Setup base style and responsive header/footer

### PR DESCRIPTION
**Pull request by lujorryn**
Create initial setup of PetPal website.

base.css:
Contains reset styles based on Josh Comeau's CSS Reset https://www.joshwcomeau.com/css/custom-css-reset/

global.css:
Styles that appear in all pages, including header, footer and default button styles.

index.html:
Implemented header and footer for home page.

about.html:
Implemented header and footer for about page, note that there's an active style for .nav__link for About Us, the active style should be propagated to all other pages.

### CODE REVIEW

File: img/logo_ref.png
Comment (JuneOh710) [inline img/logo_ref.png:1]: we should probably swap this with an SVG later but I think the png logo looks fine for now.

File: css/global.css [hunk 1]
```diff
@@ -0,0 +1,433 @@
...
+  --margin-narrow: 0.5rem;
+
+  --padding-wide: 1.5rem;
+  --padding-mid: 1rem;
+  --padding-narrow: 0.5rem;
+
+  /* type variables */
+  --type-scale: 1.25;
+  --line-height: 1.5;
+  --font-size: 1rem;
```
Comment (JuneOh710) [inline css/global.css:40]: do you think we need to define a custom default `font-size` or are we happy with just the browser default 16px?

File: css/global.css [hunk 2]
```diff
@@ -0,0 +1,433 @@
...
+:root {
+  /* color variables */
+  /* Some experiments, can delete if we are sure we don't need them */
+  /* #3dac78, #74c69d, #95d5b2, #b7e4c7, #d8f3dc */
+  /* #5603AD, #8367C7, #7EDFA0, #B3E9C7, #C2F8CB */
+  /* #344E41, #3A5A40, #588157, #5A9F68, #BBD58E */
+  /* #06a778, #c6e4b5, #eaf4cb, #f1ea9b */
+  /* #3ab284, #23c686, #c5d893 */
+  
+  --color-primary: #5a9f68;
```
Comment (JuneOh710) [inline css/global.css:14]: I like the use of css variables :)

File: img/logo_ref.png
Comment (lujorryn) [reply img/logo_ref.png:1]: Agreed, I inserted it as a placeholder for now, but we should definitely change our logo and convert to SVG later.

File: css/global.css [hunk 3]
```diff
@@ -0,0 +1,433 @@
...
+}
+
+.logo > span {
+  margin-left: var(--margin-wide);
+  color: var(--color-white);
+  font-size: var(--logo-font);
+  font-weight: 700;
+}
+
+.header__nav {
```
Comment (JuneOh710) [inline css/global.css:159]: nice naming convention!

File: css/global.css (hunk 1 above)
Comment (lujorryn) [reply css/global.css:40]: I'm not sure, but I created a variable either way in case we want to change things later it would be much faster to do so.

**Pull request closed without merging**